### ISSUE TITLE
Helium explorer missing transaction ID’s.

### ISSUE THREAD

**Issue by jas-williams** [helium/explorer#137]
Following the latest updates to the web based explorer there are no longer transaction ID's against transactions some such as assert location no longer have a click through

![image](https://user-images.githubusercontent.com/73598288/104136556-ece7ae80-538e-11eb-8b84-2c5c4ee7a573.png)

**Comment by danielcolinjames** [comment]
fixed by this commit: https://github.com/helium/explorer/commit/ab7fba6476845290eef0daec87b84aaf275b1fe2

the tags should now be clickable :)

**Comment by cascading-jox** [comment]
I noticed this issue just now. Could my cache be bad? It disappeared after some time browsing the explorer. I also noticed the hotspot is not visible in the witness list when the description is missing. This is on firefox 84. 
![Firefox_Screenshot_2021-01-16T16-09-07 465Z](https://user-images.githubusercontent.com/8418570/104816795-990d1780-581d-11eb-80b0-7397cce945b1.png)
![Firefox_Screenshot_2021-01-16T16-06-01 324Z](https://user-images.githubusercontent.com/8418570/104816742-4f243180-581d-11eb-9ef4-40d79a880f43.png)
